MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -539,10 +539,13 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
     NSString *messageFormat = details.mandatoryUpdate
                                   ? MSDistributeLocalizedString(@"MSDistributeAppUpdateAvailableMandatoryUpdateMessage")
                                   : MSDistributeLocalizedString(@"MSDistributeAppUpdateAvailableOptionalUpdateMessage");
+    NSString *appName = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:@"CFBundleDisplayName"];
+    if (!appName) {
+      appName = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:@"CFBundleName"];
+    }
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wformat-nonliteral"
```

Comment:
This is more of a question. Why are we ignoring the warning here? Is there no way to make this work. My proposal would be that we add the following to our coding "rules": Never ignore a warning by using `clang diagnostic ignored` unless there is a good reason. State that reason next to the macro so we remember why we are ignoring the warning.